crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -372,8 +372,8 @@ define_tedge_config! {
         mqtt: HostPort<MQTT_TLS_PORT>,
 
         /// Set of MQTT topics the Cumulocity mapper should subscribe to
-        #[tedge_config(example = "tedge/alarms/#,te/+/+/+/+/m/+")]
-        #[tedge_config(default(value = "te/+/+/+/+/m/+,tedge/alarms/+/+,tedge/alarms/+/+/+,tedge/events/+,tedge/events/+/+,tedge/health/+,tedge/health/+/+"))]
+        #[tedge_config(example = "tedge/alarms/#,te/+/+/+/+/m/+,te/+/+/+/+/e/+")]
+        #[tedge_config(default(value = "te/+/+/+/+/m/+,te/+/+/+/+/e/+,tedge/alarms/+/+,tedge/alarms/+/+/+,tedge/health/+,tedge/health/+/+"))]
```

Comment:
Wondering why we're **replacing** the `tedge/events` topics with the `te/.../e` topics. Shouldn't they both coexist until we fully stop supporting the `tedge/...` topics? Else who will subscribe to the `tedge/events` topics and map those messages to their `te/.../e` equivalent?

Reply:
The agent does the translation from `tedge/#` to `te/#`.

This has been implemented by https://github.com/thin-edge/thin-edge.io/pull/2125